RecoEgamma/EgammaElectronAlgos/interface/EgAmbiguityTools.h

```diff
@@ -7,7 +7,7 @@
 #include "DataFormats/GsfTrackReco/interface/GsfTrackFwd.h"
 #include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
 
-namespace EgAmbiguityTools {
+namespace electronAlgos {

```

Comment:
CMSSW style for namespaces is to use lower case. This change seems like an opportunity to conform to it.

Reply:
this is still pending

Reply:
Sorry I did not realize this was a request, I thought it was just a statement! I actually thought that just the first letter has to be lower case, but then I remember it wrong. So it should be all lower case and no spaces? That means `electronalgos`?

Reply:
would ```electron```  look OK for you, it's shorter?
perhaps even ```egamma``` can work unambiguously.

Reply:
Good suggestions! I will go for `egamma`, because some functions don't actually take electrons but other egamma objects like superclusters. Also, I think this sets a good precedent because IMHO the concept of each POG having it's namespace makes sense to me.